webapp/actions/websocket_actions.jsx

```diff
@@ -226,10 +226,18 @@ function handleEvent(msg) {
         handleReactionRemovedEvent(msg);
         break;
 
+    case SocketEvents.CHANNEL_UPDATED:
+        handleChannelUpdatedEvent(msg);
+        break;
+
     default:
     }
 }
 
+function handleChannelUpdatedEvent(msg) {
+    loadChannelsForCurrentUser(msg.broadcast.channel_id);
```

Comment:
Then here, instead of making a call back out to the server, dispatch the updated channel to the redux store:
```
const channel = JSON.parse(msg.data.channel);
dispatch({type: ChannelTypes.RECEIVED_CHANNEL, data: channel});
```